### PR TITLE
DlgCombineText

### PR DESCRIPTION
**Pull request by lilyclements**
EDIT: This is ready to review

### CODE REVIEW

**Comment by lilyclements** [comment]
@Lunalo Thank you for your comments, however, this is not yet complete. Sorry!

**Comment by lilyclements** [comment]
This dialog is currently dormant. This is because there is not an appropriate `.SetParameterTypeAs` for this yet

**Comment by stevekogo** [comment]
@dannyparsons Can you respond to @lilyclements need for .SetParameterTypeAs

**Comment by dannyparsons** [comment]
Yep this is something that has let to be implemented in the core so the dialog can't be updated yet.

**Comment by lilyclements** [comment]
@dannyparsons when shall this `.SetParameterTypeAs` be implemented in the core? This dialog on R-Instat is currently unusable since it's been half-updated and half not

**Comment by dannyparsons** [comment]
Is the dialog currently working in the master? If not then I should do this before the next release, otherwise it might have to wait.

**Comment by lilyclements** [comment]
No, it's not working in the master.
I can revert it in the master to the pre-updated code version if there isn't time to implement it for Friday, then revert back after Friday.

**Comment by dannyparsons** [comment]
Great, shall we decide on Thursday whether to revert or fix it?

File: instat/dlgCombineText.vb [hunk 1]
```diff
@@ -119,13 +107,16 @@ Public Class dlgCombineText
             lstColumnFunction = ucrReceiverCombineText.GetVariablesAsList()
             iColumnsUsed = lstColumnFunction.Count
             For i = 0 To lstColumnFunction.Count - 1
-                ucrBase.clsRsyntax.AddParameter("X" & i + 1, clsRFunctionParameter:=lstColumnFunction(i))
+                ucrBase.clsRsyntax.AddParameter("X" & i + 1, clsRFunctionParameter:=lstColumnFunction(i), iPosition:=i + 1)
```
Comment (lilyclements) [inline instat/dlgCombineText.vb]: I've gone for i+1 here because the parameter `sep` has a position of 0.

**Pull request merged**